### DISCUSSION TITLE
avatar dysphoria - 1 person, 5 avatars ?

### DISCUSSION THREAD

**Discussion by Josverl** [gitkraken/vscode-gitlens#2181] [Q&A] [answered]
what is the reason why 5 different avatars are used in the same view to describe a single person ?
1 github account , two e-mail addresses 
only once I recognize my own GH avatar, and that is credited with only a single commit 

![image](https://user-images.githubusercontent.com/981654/187859138-2c47a53b-2cd6-4e88-a8d6-7ccc63a09c5b.png)

I could understand that the two e-mail addresses or signed and unsigned accounts are treated differently , 
but this does not make sense at all 

it adds confusion, rather than context or information

if the view is rendered without the avatars, 
it is seems that the problem might not be in the avatars , bit an underlying problem in recognizing/associating/categorizing contributors 

![image](https://user-images.githubusercontent.com/981654/187861695-4bfa25c8-9678-444c-a7f4-c133cc4e6586.png)

**Comment by eamodio** [comment]
The lack of commits showing up has been fixed in 12.2.1 -- but I'm not sure what the issue is with so many "duplicates". Typically that is caused by using different emails on the commits. If you hover over the name in the Contributors view, and then hover over link on the name (or click it to open the mailto: link) you should be able to see if the emails are indeed different.

**Reply by Josverl** [reply]
ok , thanks for the fix. It indeed were multiple different spellings of the same e-mail address, and one @gihub.com address.

if only there was an option to group by name, as the reality is that one person will have multiple e-mail addresses.

**Reply by eamodio** [reply, marked as answer]
Git provides a `.mailmap` file to support that, you can read more about that here:
https://git-scm.com/docs/gitmailmap
https://blog.developer.atlassian.com/aliasing-authors-in-git/
